page/sass/common.scss

```diff
@@ -12,7 +12,7 @@ h3.smaller {
 }
 
 .section ul.compact-list {
-    padding: 0 0 $space-bottom-between-elements 0;    
+    padding: 0 0 $space-bottom-between-elements 0;
 
     li {
         font-size: 16px;
```

Comment:
And here, the number should be from the variables.scss file

Reply:
It's not part of the pull request but good observation